lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -173,6 +175,11 @@ def initialize(config_root, config_path, loaded_config)
     @data_providers = nil
   end
 
+  def fail(issue, args = EMPTY_HASH, line = nil)
```

Comment:
I think it would be better to change all `fail` as a separate ticket. Otherwise, we have a situation in between where two conventions are used.